impeller/renderer/backend/gles/reactor_gles.h

```diff
@@ -15,6 +15,8 @@
 
 namespace impeller {
 
+typedef void (*VoidCallback)(void* /* user data */);
```

Comment:
In a C++ TU, I think we should stick to using std::function and its niceties. This can just be fml::closure. The closure itself can be constructed like so in the embedder.cc

```c++
RegisterCleanupCallback(handle, [callback, user_data]() {callback(user_data);});
```

This way, you don't need to hold onto the user data baton separately either. Its just a single closure.

Reply:
Done!